### PR TITLE
Increased stack size to 128KB

### PR DESCRIPTION
**Pull request by aviy15**

### CODE REVIEW

File: Source/platform/ctr/system.cpp [hunk 1]
```diff
@@ -9,11 +9,9 @@
 
 using namespace devilution;
 
-// Increase stack size for recursion in FindTransparencyValues()
-// 128 KB supports around 500 levels of recursion
-// Default stack size on 3DS is only 32 KB
+// Increased stack size to 128KB to allow higher levels of recursion and avoid game crashes
 extern "C" {
-u32 __stacksize__ = 128 * 1024;
+u32 __stacksize__ = 4 * 128 * 1024;
```
Comment (StephenCWills) [inline Source/platform/ctr/system.cpp:14]:
```suggestion
u32 __stacksize__ = 512 * 1024;
```

File: Source/platform/ctr/system.cpp [hunk 2]
```diff
@@ -9,11 +9,9 @@
 
 using namespace devilution;
 
-// Increase stack size for recursion in FindTransparencyValues()
-// 128 KB supports around 500 levels of recursion
-// Default stack size on 3DS is only 32 KB
+// Increased stack size to 128KB to allow higher levels of recursion and avoid game crashes
```
Comment (StephenCWills) [inline Source/platform/ctr/system.cpp:12]: What is the purpose of removing the extra details in the comment? Also, you have increased the stack size to 512 KB, not 128 KB.

**Comment by AJenbo** [comment]
`8 * 128 * 1024` = 1MiB

Why? The comment is wrong, and it would probably be better written as `1024 * 1024`.

**Review by StephenCWills — CHANGES_REQUESTED** [review]
Can you provide some context for why you made this change? Have you tested this?

**Comment by qndel** [comment]
I love PRs from new guys with 0 explanation lol ...

**Pull request closed without merging**